Eatery/Controllers/BRB/BRBAccountViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  BRBAccountViewController.swift
+//  Eatery
+//
+//  Created by William Ma on 9/1/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+// MARK: -
+
+class BRBAccountViewController: UIViewController {
+    
+    private let account: BRBAccount
+    
+    private var tableView: UITableView!
+    
+    init(account: BRBAccount) {
+        self.account = account
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView = UITableView(frame: .zero, style: .grouped)
+        tableView.register(BRBBalanceTableViewCell.self, forCellReuseIdentifier: "balance")
+        tableView.register(BRBHistoryTableViewCell.self, forCellReuseIdentifier: "history")
+        tableView.allowsSelection = false
+        tableView.separatorColor = .wash
+        tableView.dataSource = self
+        tableView.delegate = self
+        view.addSubview(tableView)
+        tableView.snp.makeConstraints { make in
+            make.top.edges.equalToSuperview()
+        }
+    }
+
+}
+
+// MARK: -
+
+extension BRBAccountViewController: UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if section == 0 {
+            return 4
+        }
+        return account.history.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if indexPath.section == 0 {
+            let cell = tableView.dequeueReusableCell(withIdentifier: "balance") as! BRBBalanceTableViewCell
+
+            switch indexPath.row {
+            case 0:
+                cell.titleLabel.text = " Meal Swipes"
+                let swipesLeft = account.swipes == "" ? "Unlimited" : account.swipes
+                cell.subtitleLabel.text = swipesLeft
+            case 1:
+                cell.titleLabel.text = " BRBs"
+                cell.subtitleLabel.text = "$" + account.brbs
+            case 2:
+                cell.titleLabel.text = " Laundry"
+                cell.subtitleLabel.text = "$" + account.laundry
+            case 3:
+                cell.titleLabel.text = " City Bucks"
+                cell.subtitleLabel.text = "$" + account.cityBucks
```

Comment:
same here and for `case 2:` as well

Reply:
You can just store an array of those `titleLabel.text` and `subtitleLabel.text` values at the top and instead of having this switch statement, just directly index that array. That'll make it 10x easier to change the order or make any big structural changes to this stuff down the line.

Could also move the two text params into some struct to ensure they're being paired correctly. This may be overkill tho cc @kevinchan159

Reply:
i think the struct is a bit of an overkill